projects/observability/src/shared/dashboard/widgets/charts/cartesian-widget/cartesian-widget-renderer.component.ts

```diff
@@ -58,13 +58,17 @@ export class CartesianWidgetRendererComponent<TSeriesInterval> extends Interacti
     return this.model.getDataFetcher().pipe(
       tap(fetcher => {
         this.fetcher = fetcher;
+        const defaultInterval = this.model.defaultInterval?.getDuration();
 
         if (this.intervalSupported()) {
           this.intervalOptions = this.buildIntervalOptions();
-          this.selectedInterval = this.getBestIntervalMatch(this.intervalOptions, this.selectedInterval);
+          this.selectedInterval = this.getBestIntervalMatch(
+            this.intervalOptions,
+            this.selectedInterval ?? defaultInterval
+          );
         } else {
           this.intervalOptions = undefined;
-          this.selectedInterval = undefined;
+          this.selectedInterval = defaultInterval;
```

Comment:
If interval isn't selectable (which I'm guessing is what `intervalSupported()` represents), does this mean we're setting the interval without verifying that it's allowable for the time range?  It almost feels like this if elsecan change to something like
```ts
          const intervalOptions = this.buildIntervalOptions();
          this.selectedInterval = this.getBestIntervalMatch(
             intervalOptions,
             this.selectedInterval ?? defaultInterval
          );
          if (this.isIntervalSelectable()) {
            this.intervalOptions = intervalOptions; // The only thing this flag controls is whether options are available (and thus, the selector)
          }
```

Reply:
Yeah. Isn't that acceptable behavior? IMO, we should support setting any arbitrary interval explicitly by the user.

Reply:
I think if it is a nit issue then LGTM. we can tackle this later.

Reply:
Nope - the problem is that at high resolutions, it will put extra load on the system and/or be an invalid request. So since the system can't support it, we shouldn't here either.